phpcs-ruleset.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<!-- PHP_CodeSniffer standard -->
+<!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
+<ruleset name="Invoicing">
+    <description>DrupalDriver coding standard</description>
+
+    <rule ref="./vendor/drupal/coder/coder_sniffer/Drupal">
+    </rule>
+
+    <!-- Core drivers need to set a server's remote address. -->
+    <rule ref="Drupal.Semantics.RemoteAddress.RemoteAddress">
+        <exclude-pattern>src/Drupal/Driver/Cores/*.php</exclude-pattern>
+    </rule>
```

Comment:
In these files the `$_SERVER['REMOTE_ADDR']` predefined variable is used, which throws the following false positive:

> Use the function ip_address() instead of $_SERVER['REMOTE_ADDR']

By excluding this specific sniff in these files this false positive is suppressed.